src/localization/locales/en-US.json

```diff
@@ -498,6 +498,7 @@
         "delete_label": "Delete “{{name}}”",
         "download_label": "Download “{{name}}”",
         "invalid_extension": "The file extension ({{file.resourceType}}) does not match the file’s contents",
+        "invalid_extension_zip": "[TODO] Invalid shapefile",
```

Comment:
```suggestion
        "invalid_extension_zip": "The file {{name}} is a valid zip, but is not a valid ESRI shapefile.",
```

Reply:
@paulschreiber That is not accurate, this message could also appear if it is not a valid zip in general, so what you know here is that it is not a valid zip shapefile, this means:
1. It could be a valid zip but not a valid shape zip file
2. It could be an inavlid zip file

Reply:
Can we separate out case 1 and 2 and give different error messages for each?